static/solo/src/js/player/index.js

```diff
@@ -86,14 +87,53 @@ define(function(require) {
           player.on('tvp:media:videoended', handleEnded);
           player.on('tvp:media:ready', function(){
             playerReady = true;
+            if(player && settings && settings.hasOwnProperty("controls")){
+              $('#'+player.options.DOMContainer.id).find('.tvp-progress-bar').css('background-color', settings.controls.seekBar.progressColor);
+            }
             if ($.isFunction(callback)) {
               callback();
             }
           });
           resize();
         };
-        var defaults = require('./settings');
-        var settings = $.extend( {}, defaults, {divId:targetId });
+
+        // Check for settings is passed or not
+        if(config && config.settings){
+          if(config.settings.hasOwnProperty('progresscolor')){
+            var progresscolor = config.settings.progresscolor;
+          }else{
+            var progresscolor = '#E57211';
+          }
```

Comment:
Ah you are declaring it thrice actually, @sdorleTVPage are you looking at the Scrutinizr feedback? code didn'tpassed quality check